config/defaults.go

```diff
@@ -34,43 +34,43 @@ const (
 
 	// P2P
 	DefaultP2PPort     = 7001
-	DefaultP2PRandSeed = int64(33)
-	DefaultNoBootstrap = false
+	DefaultP2PRandSeed = int64(0)
 )
 
-var DefaultP2PBootstrap = []string{
-	"/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
-	"/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLPppuBtQSGwKDZT2M73ULpjvfd3aZ6ha4oFGL1KrGM",
-}
+var DefaultP2PBootstrap []string
```

Comment:
I assume that in the future the default will be _mainnet_ boot nodes? Should the default right now be the Alexander nodes that are listed in the `config.toml`?

Reply:
Right now I don't think we should have any bootnodes as none of them are truely compatible. Also, an opt in rather than opt out when choosing bootnodes seems more deveoper friendly right now